src/styles/components/Header.scss

```diff
@@ -0,0 +1,6 @@
+.header {
+  &__logo {
+    max-width: 12rem;
+    width: 100%;
```

Comment:
I always set `height: auto` as well, even though this technically works as-is. It just makes it more explicit what is supposed to happen.